int64 iterations
---
single_objective_path[] paths
